tests/integration/test_dataapi.py

```diff
@@ -1,44 +1,39 @@
 import unittest
 import os
 from learnosity_sdk.request import DataApi
+from learnosity_sdk.utils import Future
 
 # This test uses the consumer key and secret for the demos consumer
 # this is the only consumer with publicly available keys
 security = {
     'consumer_key': 'yis0TYCu7U9V4o7M',
     'domain': 'demos.learnosity.com'
 }
+
 # WARNING: Normally the consumer secret should not be committed to a public
 # repository like this one. Only this specific key is publically available.
 consumer_secret = '74c5fd430cf1242a527f6223aebd42d30464be22'
-request = {
+action = 'get'
+
+items_request = {
     # These items should already exist for the demos consumer
     'references': ['item_2', 'item_3'],
     'limit': 1
 }
-action = 'get'
-endpoint = '/itembank/items'
-dummy_responses = [{
-    'meta': {
-        'status': True,
-        'timestamp': 1514874527,
-        'records': 2,
-        'next': '1'
-    },
-    'data': [{'id': 'a'}]
-}, {
-    'meta': {
-        'status': True,
-        'timestamp': 1514874527,
-        'records': 2
-    },
-    'data': [{'id': 'b'}]
-}]
+items_endpoint = '/itembank/items'
+
+questions_request = {
+    # These items should already exist for the demos consumer
+    'item_references': ['py-sdk-test-2019-1', 'py-sdk-test-2019-2'],
+    'limit': 1
+}
+questions_endpoint = '/itembank/questions'
 
 
 class IntegrationTestDataApiClient(unittest.TestCase):
 
-    def _build_base_url(self):
+    @staticmethod
+    def __build_base_url():
```

Comment:
I changed this to a static method as it's only used internally by the integration tests and does not access any class variables.